examples/basic_scan_progress/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"github.com/Ullaakut/nmap/v2"
+	"log"
```

Comment:
Please separate the stdlib imports and external/internal imports

```suggestion
	"fmt"
	"log"
	
	"github.com/Ullaakut/nmap/v2"
```